slick/src/main/scala/slick/jdbc/PositionedResult.scala

```diff
@@ -32,6 +32,7 @@ abstract class PositionedResult(val rs: ResultSet) extends Closeable { outer =>
   final def <<? [T](implicit f: GetResult[Option[T]]): Option[T] = if(hasMoreColumns) this.<< else None
 
   final def nextBoolean()    = { val npos = pos + 1; val r = rs getBoolean    npos; pos = npos; r }
+  final def nextChar()       = { val npos = pos + 1; val r = rs getString     npos; pos = npos; if(r eq null) null else r.charAt(0) }
```

Comment:
null looks weird here, all other types except for BigDecimal don't have that. Why is that?

Also I am worried that this just discards all further characters. Should we add `assert( r.size == 1, "found multiple characters in supposed char column: "+r )`